docs/src/man/getting_started.md

```diff
@@ -476,6 +476,49 @@ julia> df[in.(df.A, Ref([1, 5, 601])), :]
 
 Equivalently, the `in` function can be called with a single argument to create a function object that tests whether each value belongs to the subset (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+#### Column selection syntax
+
+You can also use the [`select`](@ref) and [`select!`](@ref) functions to select columns in a data frame.
```

Comment:
How about mentioning the main differences between `select` and indexing, namely: 1) that `select(df, :x1)` returns a data frame rather than a vector? Also, `select` makes copies like `df[:, cols]`, which isn't obvious.

Reply:
added